.github/workflows/windows-build-xpu-nightly.yml

```diff
@@ -0,0 +1,102 @@
+name: windows-build-xpu-nightly
```

Comment:
Let's delete this workflow and focus on ci firstly, the CD has a series dedicate scripts and workflows